### DISCUSSION TITLE
Thank you for restoring my feed!

### DISCUSSION THREAD

**Discussion by dmadisetti** [community/community#37900] [Other Feature Feedback, Questions, & Ideas]
Thank you for restoring my feed! Not sure whether this is because I wrote to support, or what- but I am super grateful that you restored the feed.

I just want to provide a bit of feedback regarding. The new feed _must have_:

- and option for Chronological sorting
- and option the same content as the current feed, with the option to add in the additional information.

Updates from collaborators, related projects _must be optional_. I find my feed is **already** too noisy. In fact, if you gave me the option to _mute_ my followers (or particular actions such as starring/ forking), I would likely follow more people.

**Comment by ettaboyle** [comment]
@dmadisetti Thank you for sharing this update and your feedback. It's really helpful! 
I've added the `product feedback` label to be sure this is shared with the engineering team.

**Reply by boardgamesaz** [reply]
Nice and interesting post. Thank You! For sharing such a great article. If you can't access your account and imagine you forgot your password and you don't know what to do then I know one blog about the <a href="https://poppyplaygame.io/">poppy playtime</a> and get back to your account.

**Comment by jfullstackdev** [comment]
at the `For You` there is already the filter 

![image](https://user-images.githubusercontent.com/47092464/199153081-d280b6a9-b81f-413c-b74f-4eb8b965120d.png)

the `Following` is the older version of the feed

![image](https://user-images.githubusercontent.com/47092464/199153134-d28833eb-985b-4f08-bc63-3cdda800fe28.png)

also, if you want to focus more on your development, you can create an Org and create repos there, there is a dedicated dashboard all about the Org really, not missing anything all about those repos, like this 

![image](https://user-images.githubusercontent.com/47092464/199153253-872cf6e3-5bc6-4e91-9012-1cb4d6656509.png)

@ettaboyle I've been seeing a lot of those request, maybe what they are asking is that, there is a filter for only all the watched repos and their repos only, like the Org dashboard, a filter for `Discover` etc., like adding all of the possible feeds on the filter but can be filtered so they can focus, as of now, there is no dedicated one for the Dashboard of all the watched repos and their personal repos activities, also a feed when someone starred your repo and started following you, it was in the past but no longer there in the new one I think so

**Comment by dmadisetti** [comment]
I was in an experiment that removed the following page.

Also, the recommendation filter for "For You" is still too noisy and not useful
